dockerfiles/kotlin-2.0.Dockerfile

```
@@ -0,0 +1,13 @@
+# syntax=docker/dockerfile:1.7-labs
+FROM maven:3.9.8-eclipse-temurin-22-alpine
+
+# Ensures the container is re-built if pom.xml changes
+ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="pom.xml"
+
+WORKDIR /app
+
+# .git & README.md are unique per-repository. We ignore them on first copy to prevent cache misses
+COPY --exclude=.git --exclude=README.md . /app
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)